SwiftPackageManager.md

```diff
@@ -11,8 +11,8 @@ Package Manager](https://swift.org/package-manager/) in Beta status.
 - Requires Xcode 12.
 - Analytics requires clients to add `-ObjC` linker option.
 - Analytics is only supported for iOS and cannot be used in apps that support other platforms.
-- Performance and Firebase ML are not currently available.
-- watchOS support is introduced in 7.0.0 for Auth, Crashlytics, Messaging, and Storage.
+- Performance is not yet available.
+- watchOS support is available for Auth, Crashlytics, Messaging, RemoteConfig, and Storage.
```

Comment:
ABTesting as well?

Reply:
Probably not, since it's not usable on it's own yet without Analytics?